Makefile

```diff
@@ -75,16 +74,21 @@ test:
 	    color=$${RED}; \
 	    message=''; \
 	fi; \
-	echo -e "\n\tTest $${color}$${status}$${NC}. Code=$${code}$${message}\n"; \
+	echo -e -n "\n\tTest "; \
+	tput setaf $${color}; \
+	tput bold; \
+	echo -e -n "$${status}"; \
+	tput sgr0; \
+	echo -e ". Code=$${code}$${message}\n"; \
 	rm -f $(example10_data_dir)/{ex10_res_int_se.dat,ex10_res_int_prop_se.dat,ex10_res_int_prop_me.dat}; \
 	if [ $$remove_tmp_dir -eq 1 ]; then \
 	    cd $(root_dir); \
 	    rm -rf $(example10_data_dir); \
 	fi;
 
 clean:
-	rm -f $(ext_dir)/libfftw.a $(gcc_dir)/libsrw.a $(gcc_dir)/srwlpy.so; \
+	rm -f $(ext_dir)/libfftw.a $(gcc_dir)/libsrw.a $(gcc_dir)/srwlpy*.so; \
 	rm -rf $(ext_dir)/$(fftw_dir)/ py/build/;
-	if [ -d $(root_dir)/.git ]; then git checkout $(examples_dir)/srwlpy.so; fi;
+	if [ -d $(root_dir)/.git ]; then rm -f $(examples_dir)/srwlpy*.so && (git checkout $(examples_dir)/srwlpy*.so 2>/dev/null || :); fi;
```

Comment:
Done, placed in `\\nsls2expdev1\rakitin\recent_developments\2017-01-27 SRW new developments\`.